app/src/src_nonfreenet/org/breezyweather/sources/lhmt/LhmtResultConverter.kt

```diff
@@ -0,0 +1,531 @@
+        "light-snow-at-times" -> context.getString(R.string.common_weather_text_snow_light)
+        "snow-at-times" -> context.getString(R.string.common_weather_text_snow)
+        "light-snow" -> context.getString(R.string.common_weather_text_snow_light)
+        "snow" -> context.getString(R.string.common_weather_text_snow)
+        "heavy-snow" -> context.getString(R.string.common_weather_text_snow_heavy)
+        "snowstorm" -> context.getString(R.string.common_weather_text_snow_heavy)
+        "fog" -> context.getString(R.string.common_weather_text_fog)
+        "squall" -> context.getString(R.string.common_weather_text_squall)
+        else -> null
+    }
+}
+
+private fun getWeatherCode(
+    code: String?,
+): WeatherCode? {
+    return when (code) {
+        "clear" -> WeatherCode.CLEAR
+        "partly-cloudy" -> WeatherCode.PARTLY_CLOUDY
+        "variable-cloudiness" -> WeatherCode.PARTLY_CLOUDY
+        "cloudy-with-sunny-intervals" -> WeatherCode.PARTLY_CLOUDY
+        "cloudy" -> WeatherCode.CLOUDY
+        "rain-showers" -> WeatherCode.RAIN
+        "light-rain-at-times" -> WeatherCode.RAIN
+        "rain-at-times" -> WeatherCode.RAIN
+        "light-rain" -> WeatherCode.RAIN
+        "rain" -> WeatherCode.RAIN
+        "heavy-rain" -> WeatherCode.RAIN
+        "thunder" -> WeatherCode.THUNDER
+        "isolated-thunderstorms" -> WeatherCode.THUNDERSTORM
+        "thunderstorms" -> WeatherCode.THUNDERSTORM
+        "heavy-rain-with-thunderstorms" -> WeatherCode.THUNDERSTORM
+        "sleet-showers" -> WeatherCode.SLEET
+        "sleet-at-times" -> WeatherCode.SLEET
+        "light-sleet" -> WeatherCode.SLEET
+        "sleet" -> WeatherCode.SLEET
+        "freezing-rain" -> WeatherCode.SLEET
+        "hail" -> WeatherCode.HAIL
+        "snow-showers" -> WeatherCode.SNOW
+        "light-snow-at-times" -> WeatherCode.SNOW
+        "snow-at-times" -> WeatherCode.SNOW
+        "light-snow" -> WeatherCode.SNOW
+        "snow" -> WeatherCode.SNOW
+        "heavy-snow" -> WeatherCode.SNOW
+        "snowstorm" -> WeatherCode.SNOW
+        "fog" -> WeatherCode.FOG
+        "squall" -> WeatherCode.WIND
+        else -> null
+    }
+}
+
+private fun getAlertText(
+    context: Context,
+    text: LhmtAlertText?,
+): String? {
+    return if (context.currentLocale.code.startsWith("lt")) {
+        text?.lt
+    } else {
+        text?.en
+    }
+}
+
+private fun getAlertSeverity(
+    severity: String?,
+): AlertSeverity {
+    return with(severity) {
+        when {
+            equals("Extreme", ignoreCase = true) -> AlertSeverity.EXTREME
+            equals("Severe", ignoreCase = true) -> AlertSeverity.SEVERE
+            equals("Moderate", ignoreCase = true) -> AlertSeverity.MODERATE
+            equals("Minor", ignoreCase = true) -> AlertSeverity.MINOR
+            else -> AlertSeverity.UNKNOWN
+        }
+    }
+}
+
+private val MUNICIPALITIES = listOf(
```

Comment:
Please document how to get the values below

Reply:
Fixed.